content/docs/quickstart/kubernetes/_index.md

```diff
@@ -6,12 +6,32 @@ menu:
     identifier: kubernetes
     name: Kubernetes
     weight: 2
+cloudlink: /docs/reference/clouds/kubernetes
 ---
+Learn how to quickly provision infrastructure on Kubernetes with Pulumi using JavaScript, TypeScript, or Python as your language runtime.
 
-This guide is intended to help you quickly get started provisioning infrastructure on Kubernetes with Pulumi.
+Pulumi's Cloud Native SDK makes it easy to target any Kubernetes environment to provision a cluster, configure and deploy applications, and update them as required.
 
-<!-- TODO embellish the summary/overview of the guide. -->
+Pulumi works with Minikube, on-premises and cloud-hosted custom Kubernetes clusters, and the managed services from Google (GKE), Azure (AKS), and Amazon (EKS).
+
+<div class="bg-transparent border border-gray-300 hover:bg-gray-200 mb-5 flex justify-center">
+    <ul class="flex list-none">
+        <li>
+            <img class="h-12 mr-6" src="/images/logo/logo-symbol.svg" alt="Pulumi logo">
+        </li>
+    </ul>
+    <ul class="flex list-none mt-8 p-4">
+        <li><img class="h-8 ml-6 mr-6" src="/logos/tech/logo-minikube.png" alt="MiniKube" title="MiniKube"></li>
+        <li><img class="h-8 mx-6" src="/logos/tech/logo-kubernetes.png" alt="Kubernetes" title="Kubernetes"></li>
+        <li><img class="h-8 mx-6" src="/logos/tech/logo-aws.png" alt="AWS" title="Amazon Web Services"></li>
+        <li><img class="h-8 mx-6" src="/logos/tech/logo-azure.png" alt="Azure" title="Azure"></li>
+        <li><img class="h-8 mx-6" src="/logos/tech/logo-gd.png" alt="GCP" title="Google Cloud Platform"></li>
+    </ul>
+</div>
+
+{{< essential-links "Kubernetes" >}}
```

Comment:
Since this _is_ the getting-started flow, I'm not sure we should offer such an immediate departure. Would this be better placed somewhere further down the line? Like toward the end?

Reply:
Hmm, let me think more on this. Might be a good candidate for other landing pages (maybe even the docs homepage), but not the Get Started flow. So, another thing I'm thinking of on the topic of styling: If we were to [update the docs index page to a two-column layout](https://github.com/pulumi/docs/issues/1325), where do you see the "Search" box moving to? Will it be too cluttered on the left nav?

Reply:
> If we were to update the docs index page to a two-column layout, where do you see the "Search" box moving to? Will it be too cluttered on the left nav?

We could probably make it work, or situate it somewhere else. Sounds like a good thing to whiteboard through maybe.